roles/ceph-common/defaults/main.yml

```diff
@@ -244,6 +244,7 @@ rbd_client_admin_socket_path: /var/run/ceph # must be writable by QEMU and allow
 monitor_interface: interface
 monitor_address: 0.0.0.0
 mon_use_fqdn: false # if set to true, the MON name used will be the fqdn in the ceph.conf
+monitor_address_block: false
```

Comment:
That's a feature that was added to the ceph.conf. However, it fails on an OSD to create the ceph.conf unless this is defaulted.